spec/mailboxes/comments_mailbox_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe CommentsMailbox, type: :mailbox do
+  subject { process(mail) }
+
+  let(:ticket) { create(:ticket) }
+  let(:support_email) { "ticket-#{ticket.id}@#{ApplicationMailer::SUPPORT_DOMAIN}" }
```

Comment:
Why is the email address' username have the ticket-{ticketNumber} format? Is the app validating this format somehow?